docs/extras/use_cases/extraction/index.mdx

```diff
@@ -12,13 +14,106 @@ Examples of this include:
 - Extracting multiple rows to insert into a database from a long document
 - Extracting the correct API parameters from a user query
 
-This work is extremely related to [output parsing](/docs/modules/model_io/output_parsers/).
-Output parsers are responsible for instructing the LLM to respond in a specific format.
-In this case, the output parsers specify the format of the data you would like to extract from the document.
-Then, in addition to the output format instructions, the prompt should also contain the data you would like to extract information from.
+## Overview
+
+In LangChain we provide a few useful abstractions that help you leverage OpenAI [function calling](https://openai.com/blog/function-calling-and-other-api-updates) (`-0613` models) and which go a long way to avoid model hallucination. This is in general the best go-to to extract structured data from text using langchain. However there's other two options that you should also consider:
```

Comment:
Here we might offer a simple [schematic](https://docs.google.com/presentation/d/1zHk0Wwmvl_RM3FkiBZaOrts0cVoPHCcT0s6cmSwnSdo/edit?usp=sharing) to explain the differences:

![image](https://github.com/langchain-ai/langchain/assets/122662504/4640142c-b45b-48c4-98e0-29514de6d4d6)